### PR TITLE
When reading pypirc config, read setup.cfg as well to override this.

### PR DESCRIPTION
**Pull request by mauritsvanrees**
Fixes issue #436.

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
@wesleybl can you try this?

**Comment by mauritsvanrees** [comment]
Note: I wondered about reading `pyproject.toml` as well, but it does not look like `twine` itself is reading this, at least not for the distutils/index-servers setting.

**Review by reinout — APPROVED** [review]
One comment on a possible wording change.

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,8 @@ Changelog for zest.releaser
 9.1.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- When reading ``~/.pypirc`` config, read ``setup.cfg`` as well to override this.
```
[the remark this answers is not in the document]
Comment (mauritsvanrees) [reply CHANGES.rst]: Thanks.  I reworded it slightly differently.

**Comment by wesleybl** [comment]
> @wesleybl can you try this?

@mauritsvanrees I tested it and it worked. Thanks!

**Pull request merged**